CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 2.6.0-beta1
+- [Add support for Heartbeat properties in AspNetCore. Add App Services and Azure Instanace Metedata heartbeat provider modules by default, support disable of heartbeats.](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/627)
```

Comment:
Metadata - spellcheck.
Add support for Heartbeat properties - we already had this support right? So just mention about the ability to turn-off hearbeat.